src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -7,37 +7,38 @@ import ch.qos.logback.classic.Level
 import ch.qos.logback.classic.pattern.ThrowableProxyConverter
 import ch.qos.logback.classic.spi.ILoggingEvent
 import ch.qos.logback.core.LayoutBase
-import cromwell.logging.TerminalLayout.EnhancedILoggingEvent
 import cromwell.util.TerminalUtil
 
+object TerminalLayout {
+  val Converter = new ThrowableProxyConverter
+  Converter.start()
+
+  implicit class EnhancedILoggingEvent(val event: ILoggingEvent) extends AnyVal {
+    def toStackTrace: String = Converter.convert(event)
+  }
+
+  implicit class ColorString(msg: String) {
+    def colorizeUuids: String = {
+      "UUID\\((.*?)\\)".r.findAllMatchIn(msg).foldLeft(msg) {
+        case (l, r) =>
+          val color = (Math.abs(17 * r.group(1).substring(0,8).map(_.toInt).product) % 209) + 22
+          l.replace(r.group(0), TerminalUtil.highlight(color, r.group(1)))
+      }
+    }
+    def colorizeCommand: String = msg.replaceAll("`([^`]*?)`", TerminalUtil.highlight(5, "$1"))
+  }
+}
+
```

Comment:
There's not a reliable way to do that because colors are defined by the terminal emulator.  We basically just choose a number 0-255 and your terminal emulator chooses a color for each of those 256 values and that's based on the theme you have set up.  There's only vague guarantees for colors 0-8 and even then they vary quite a bit.  For all default color output, we DO use the 0-8 values, and the RAINBOW_UUID option will open that up to 256 colors so each UUID can have its own color without too many collisions.